### PR TITLE
GoogleDrive - add option to use created date as modified date

### PR DESCRIPTION
**Pull request by buchanan**

### CODE REVIEW

**Comment by ncw** [comment]
What scenarios do you see this being useful in? Can you patch the docs too `docs/content/drive.md` with a description of the flag and why you might want to use it too?

Using the created date will break `rclone sync` unless you use `--checksum` I think.

**Comment by ncw** [comment]
According to travis you need to run `go fmt` on your code (or [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports))

**Comment by buchanan** [comment]
Maybe my use case is specific and doesn't warrant an additional feature/flag. Also maybe some additional logic should be added. I'll explain.
I wanted to download my photos from google and this seemed like the perfect tool. I've used it in the past with no problems. However after enabling the put my Photos in my Google Drive option all of my photos modified dates were set to today. In order to retain the date that the photo/video was originally taken on I need to grab the creation date. This may only be useful when moving data down to your local pc as I don't believe it's possible to set the creation date via the drive API.
Would it be possible to only check this flag when the source is a google drive?

File: docs/content/drive.md [hunk 1]
```diff
@@ -293,6 +293,12 @@ Controls whether files are sent to the trash or deleted
 permanently. Defaults to true, namely sending files to the trash.  Use
 `--drive-use-trash=false` to delete files permanently instead.
 
+#### --drive-use-created-date ####
+
+Use the file creation date inplace of the modification date. Defaults to false.
+Useful when moving data the your local fs and you want to use the upload/creation date
```
Comment (ncw) [inline docs/content/drive.md]: There is a typo here... "moving data the your local fs"

Can you add a bit about the google photos too - I thought that was interesting.

**Comment by ncw** [comment]
Thanks for the explanation - I spotted a typo in the docs - can you add the info about google photos too then I'll merge - thanks :-)

**Comment by ncw** [comment]
Sorry, I forgot to merge this PR and in the mean time we upgraded the drive API from v2 to v3 which it means it doesn't merge any more.

So, since it is my fault, I've rebased and fixed up this PR manually in 9aa88159904706a8477f7a963b87d4b1cbbe4c8e and it is now merged.

Thank you very much :-)

**Pull request closed without merging**